package.json

```diff
@@ -103,9 +103,9 @@
 		"@hpi-schul-cloud/commons": "^1.3.4",
 		"@keycloak/keycloak-admin-client": "^21.0.0",
 		"@lumieducation/h5p-server": "^9.2.0",
-		"@mikro-orm/core": "^5.4.2",
-		"@mikro-orm/mongodb": "^5.4.2",
-		"@mikro-orm/nestjs": "^5.1.5",
+		"@mikro-orm/core": "^5.5.3",
```

Comment:
can we try latest version instead?

Reply:
Yes, once we will get to the 5.5.x version we can try some higher ones, at least to the point the tests will still be passing.

Reply:
@virgilchiriac I've tried to update the `mikro-orm` dependencies to higher versions than the ones mentioned in the ticket (I've tried 5.7.x and even 5.6.x), but unfortunately I did not succeed in that - there appeared some new errors which I didn't have time to investigate. But thankfully @uidp offered his help in the further work on these `mikro-orm` dependencies updates so please feel free to create another ticket for some higher versions update, this PR will not include it as it's blocking some ticket that needs an update to the 5.5.x version.